disco_aws_automation/disco_storage.py

```diff
@@ -131,6 +175,10 @@ def is_ebs_optimized(self, instance_type):
         """Returns true if the instance type is EBS Optimized"""
         return instance_type in EBS_OPTIMIZED
 
+    def is_ebs_encrypted(self, instance_type):
```

Comment:
Maybe something like `supports_ebs_encryption` might be better. My first thought after reading `is_ebs_encrypted` was that it would take a ebs volume and return if its encrypted. This method takes a instance type and returns if it supports ebs encryption.